Source/Provers/SMTLib/SMTLibInteractiveTheoremProver.cs

```diff
@@ -224,7 +224,7 @@ public override void FullReset(VCExpressionGenerator generator)
               {
                 usedNamedAssumes.Add(resp.Name);
                 if (libOptions.TrackVerificationCoverage) {
-                  reporter.AddCoveredElement(new LabeledNodeComponent(resp.Name.Substring("aux$$assume$$".Length)));
+                  reporter.AddCoveredElement(TrackedNodeComponent.ParseSolverString(resp.Name.Substring("aux$$assume$$".Length)));
```

Comment:
I take it these two lines were the fundamental bug?

Reply:
Yep.